app/views/pets/_pet.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.extract! pet, :id, :created_at, :updated_at
```

Comment:
Do you know what these `.jbuilder` files are? I would assume they aren't hurting anything, but wondering why we need them.

Reply:
Best answer i found for these `json.bulder` files:
`As said before, Rails 4 seems to generate the files as a way to have a template for responding to a JSON query. So let's say you have "scaffolded" a model Car, then it will respond with a HTML page and a detail view of car (id 1) if you go /car/1.

Then if you go to /car/1.json, it will render the show.json.jbuilder file. There also seems to be some kind of automatic mechanism as the show method is empty when scaffolding`

Do not think we'll need them especially since the article I found on them was from 2013 and neither of us has seen them before.